org-code-javabuilder/lib/src/main/java/dev/javabuilder/CORSFilter.java

```diff
@@ -1,81 +0,0 @@
-package dev.javabuilder;
```

Comment:
I think CORSFilter gets used by the WebSocketServer version of javabuilder in order to provide prompter functionality and to get the theater files to the client.

Reply:
oh good call out, will add back in